lib/index.js

```diff
@@ -162,16 +170,28 @@ export default class HyperList {
       }
     })
 
-    // Decorate the container element with styles that will match
+    // If using React, the element object turns into a props object.
+    if (config.isReact) {
```

Comment:
Yup, I agree. For now this allows it to work so I can move forward, but I don't think we should merge this as-is.